content/install_maintain/installation/installing-cluster.md

```diff
@@ -221,24 +179,93 @@ postgresql_server:
         # Should be the same on all nodes
         postgres:
             replicator_password: '<strong password for replication user>'
+  
 
-    enable_remote_connections: true
-    ssl_enabled: true
-    ssl_only_connections: true
+# For monitoring service(status reporter)
+prometheus:
+  credentials:
+    username: <monitoring username>
+    password: <srtong password for monitoring user>
 
-    # If true, client certificate verification will be required for PostgreSQL clients,
-    # e.g. Cloudify Management service cluster nodes.
-    ssl_client_verification: false
+  cert_path: <certificate for prometheus, cert_path of postgresql_server can be used>
+  key_path: <key for promethus, key_path of postgresql_server can be used>
+  ca_path: <ca for promethus, key_path of postgresql_server can be used>
+  postgres_exporter:
+    # `password` is a placeholder and will be updated during config file rendering, based on postgresql_server.postgres_password
+    password: ''
+    sslmode: require
```

Comment:
This should change soon, once CY-3266 is dealt with.

Reply:
Lets add task to CY-3266 to fix docs after done